qsirecon/workflows/recon/anatomical.py

```diff
@@ -807,46 +818,6 @@ def _get_resampled(atlas_configs, atlas_name, to_retrieve):
     return atlas_configs[atlas_name][to_retrieve]
 
 
-def get_t1w_registration_node(infant_mode, sloppy, omp_nthreads):
```

Comment:
Does this not get used anywhere? I think I added it as part of a plan to enable spatial normalization during the recon workflows if one was missing but may have not used it anywhere

Reply:
It gets called in some commented-out code, but I think we'll take a different tack with the spatial normalization (i.e., output-spaces and whatnot).